addons/website_form/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -13,8 +13,8 @@
                     width: 100%;
                 }
                 td {
-                    &:first-child, &:last-child {
-                        width: 28px;
+                    &:first-child, &:nth-child(3), &:last-child {
+                        width: 22px;
                     }
                 }
```

Comment:
Using `table-layout: auto` and adding some horizontal padding to the cells will be more natural instead of hardcoding